src/Mod/Part/App/TopoShape.cpp

```diff
@@ -372,20 +372,19 @@ void TopoShape::convertToMatrix(const gp_Trsf& trsf, Base::Matrix4D& mtrx)
 #if OCC_VERSION_HEX >= 0x070000
     gp_Mat m = trsf.VectorialPart();
     gp_XYZ p = trsf.TranslationPart();
-    Standard_Real scale = trsf.ScaleFactor();
 
     // set Rotation matrix
-    mtrx[0][0] = scale * m(1,1);
-    mtrx[0][1] = scale * m(1,2);
-    mtrx[0][2] = scale * m(1,3);
+    mtrx[0][0] = m(1,1);

```

Comment:
gp_Trsf::VectorialPart returns the 3x3 matrix including the scale factor now [[1]](https://www.opencascade.com/doc/occt-7.0.0/refman/html/classgp___trsf.html#aa78be9e0b993ddd58f0ea6073a1ecf16), an equivalent change to me removing the (redundant) scale factor multiplication here would be to change to gp_Trsf::HVectorialPart [[2]](https://www.opencascade.com/doc/occt-7.0.0/refman/html/classgp___trsf.html#a314df1793e0b41f74bf7cf3c35284ef0) which does not include the scale factor.